tox.ini

```diff
@@ -74,7 +74,8 @@ commands =
     {toxinidir}/scripts/run_regression.sh
 deps =
     unittest2
-#   Use a development checkout of oauth2client until a release is made
-#   which fixes https://github.com/google/oauth2client/issues/125
-    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
Don't we need to call out `oauth2client` as an explicit dependency (pulled from PyPI, rather than checked out)?